index.js

```diff
@@ -202,7 +202,7 @@ const baseOpen = async options => {
 		}
 
 		if (appArguments.length > 0) {
-			appArguments = appArguments.map(arg => `"\`"${arg}\`""`);
+			appArguments = appArguments.map(arg => `"\`"${String.prototype.replaceAll.call(arg, '"', '`"')}\`""`);
```

Comment:
```suggestion
			appArguments = appArguments.map(arg => `"\`"${String(arg).replaceAll('"', '`"')}\`""`);
```

Reply:
This does not handle the case where the quote is already escaped: `\"`

Reply:
Sorry for the late response. Could you please elaborate on the `\"` escaping? I can't find anything on using `\` as an escape character in PowerShell.

Alternatively, would it make sense to use single quote strings, as they don't interpret special characters? You can include single quotes in single quote strings by doubling them. This would almost certainly eliminate any edge cases.